### PR TITLE
Review navbar buttons and header toggle

### LINKED ISSUE #19: Sprint 1 & 2: Add Decals and Sections to NavBar
# Add in the Buttons for Sections to the NavBar section (on the left).

This will mean creating the css of the navbar, creating each of the sections; myDocuments, Browse, Create, Upload ect.

### LINKED ISSUE #45: Sprint 2: Add toggle functionality to Header buttons
This involves adding a toggle behaviour to the _Save all_ and _Validate all_ buttons. Label text should change to _Saved_ and _Validated_ on click.

### PR DESCRIPTION
**Pull request by max-romagnoli**
Resuming the review from PR #58.

This will close #19 and close #45.

### CODE REVIEW

**Comment by max-romagnoli** [comment]
image

**Comment by max-romagnoli** [comment]
image

**Comment by max-romagnoli** [comment]
image

File: client/src/components/Navbar.js [hunk 1]
```diff
@@ -5,22 +5,14 @@ import styles from './Navbar.module.css'
 function Navbar() {
   return (
```
Comment (max-romagnoli) [inline client/src/components/Navbar.js:6]: Passed icon svg as prop to allow for restyling.

File: client/src/icons/file.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" standalone="no"?>
```
Comment (max-romagnoli) [inline client/src/icons/file.svg:1]: Added css attribute to make svg editable.

**Review by MaxCunningham19 — APPROVED** [review]
LGTM

**Review by ciannamacmahon — APPROVED** [review]
looks good

**Pull request merged**